crates/cheatnet/src/runtime_extensions/forge_runtime_extension/cheatcodes/spoof.rs

```diff
@@ -21,43 +21,12 @@ pub struct TxInfoMock {
 
 impl CheatnetState {
     #[allow(clippy::too_many_arguments)]
-    pub fn start_spoof(
-        &mut self,
-        target: CheatTarget,
-        version: Option<Felt252>,
-        account_contract_address: Option<Felt252>,
-        max_fee: Option<Felt252>,
-        signature: Option<Vec<Felt252>>,
-        transaction_hash: Option<Felt252>,
-        chain_id: Option<Felt252>,
-        nonce: Option<Felt252>,
-        resource_bounds: Option<Vec<Felt252>>,
-        tip: Option<Felt252>,
-        paymaster_data: Option<Vec<Felt252>>,
-        nonce_data_availability_mode: Option<Felt252>,
-        fee_data_availability_mode: Option<Felt252>,
-        account_deployment_data: Option<Vec<Felt252>>,
```

Comment:
I think the main reason it was done this way is that in cheatcodes we take `"cairo-like"` values. I don't think this refactor is bad but just wanted to point it